libs/templates/src/lib/home/index.tsx

```diff
@@ -87,6 +88,62 @@ export function HomeTemplate() {
       </Hero.ContentLimitless>
 
       <StatBlock.NumbersOnly cardsContent={homeStatNumbers} />
+      <Features.ContentLeft
+        title="Fast and reliable"
+        content={
+          <ul className="flex max-w-[500px] flex-col gap-gap-xl">
```

Comment:
this `max-w-[500px]` is causing issues on mobile version, please don't use this. I think the `ul` should full width.